content/docs/payments/verify-payments.mdx

```diff
@@ -0,0 +1,49 @@
+---
+title: Verify Payments
+description: Verify Payments page
+---
+
+# Verify Payments
+
+Transactions are being verified using the Verify Payments endpoint from your server using your transaction reference or with the address property whose value is the payment address returned from the <a href="#">initialise payment</a> function.
+
+## Verify Payment API
+
+To verify transactions using the Lazerpay endpoint, you will make a <code>get</code> request to verify transaction endpoint from your server using your transaction reference or with the address property.
+
+Here’s a code sample for verifying transactions.
+
+<LzTable 
+  reverse
+  head={['Parameter name', 'Type', 'Description']}
+  body={[
+    ['<strong>Path</strong>', '', ''],
+    ['address_or_reference', '<code>String</code>', 'The transaction <code>Address</code> or <code>reference</code> you want to fetch'],
+    ['<strong>Header</strong>', '', ''],
+    ['*', '<code>String</code>', 'YOUR_PUBLIC_KEY'],
+  ]}
+  />
+
+<CodeBlock responseTheme="red" responseTitle="Response" data={{type: 'request', item: {method: "GET", requestData: {name: ['cURL', 'Node'], snippet: ['verifyPaymentGetCURL', 'verifyPaymentGetCURL'], lang: ['curl', 'curl']}, responseData: {name: ['200: OK', '404: Not Found'], snippet: ['verifyPaymentGetResponse200', 'verifyPaymentGetResponse404'], lang: ['json', 'json']}} }} />
+
+#### Important Note from the verification response
+1. The <code>actualAmount</code> key is the amount that you intended to charge the customer in crypto.
+2. The <code>amountPaid</code> key is the crypto amount that is being paid to the merchant. It is usually in the coins that we support..
+3. The <code>amountPaidFirst</code> is the fiat value of the crypto amount beinf paid to the merchant.
+4. The <code>fiatAmount</code> is the fiat value of the amount that you intended to charge the customer. This is usually in the merchant’s local currency.
+
+<Alert>The <code>amountReceivedFiat</code> and <code>amountReceived</code> values is the amountPaidFiat plus Lazerpay fees and the AmountPaid plus Lazerpay fees respectively</Alert>
+
+<br />
+
+## From Official JS SDK
+With the JS SDK, you have access to the confirmPayment object which is used to verify any payment in the Lazerpay platform.
+
+<CodeBlock responseTheme="red" responseTitle="JSON" data={{type: 'request', item: { requestData: {name: ['Node'], snippet: ['verifyPaymentNodeSDK'], lang: ['js']}, responseData: {name: ['200: OK', '401: Unauthorized'], snippet: ['customerInfoReq201', 'customerInfoReq401'], lang: ['json', 'json']}} }} />
+
+<br />
+<Alert status="warning">If you offer digital value like airtime, wallet top-up, digital credit, etc, always confirm that you have not already processed the value for that transaction to avoid double fulfillments, especially, if you also use webhooks.</Alert>
+<Divider className='max-w-100 mt-1 mb-8' />
+<br />
+<Divider />
+<Pagination />
```

Comment:
Please remove this pagination component. This should only exist at the Layout level